pkg/client/devops/jenkins/internal/gitlab.go

```diff
@@ -86,6 +86,7 @@ func AppendGitlabSourceToEtree(source *etree.Element, gitSource *devopsv1alpha3.
 		regexTraits.CreateAttr("plugin", "scm-api")
 		regexTraits.CreateElement("regex").SetText(gitSource.RegexFilter)
 	}
+	traits.CreateElement("io.jenkins.plugins.gitlabbranchsource.GitLabSkipNotificationsTrait")
```

Comment:
I prefer to provide an option for this. So, users are able to choose if skip the notification from Jenkins by themself. We could make this to be the default behavior.

For example, `gitSource.SkipJenkinsNotification`.

Reply:
Yes, good advice.

Reply:
@LinuxSuRen  I want define `SkipJenkinsNotification` field is `bool`, but `bool` default is `false`.  
Change field name `AcceptJenkinsNotification` ?

Reply:
Fine for me.

Reply:
By the way, please don't forget to add the unit tests.